packages/sdk/src/clients/etherchain/client.ts

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+import { get } from "lodash";
+
+type GasPrice = {
+  safeLow: number;
+  standard: number;
+  fast: number;
+  fastest: number;
+  currentBaseFee: number;
+  recommendedBaseFee: number;
+};
+
+export class Etherchain {
+  private url: string;
+
+  constructor(url = "https://www.etherchain.org/api") {
```

Comment:
You can shortcut defining the private var by saying 'private url' here and remove the declaration of the var above